#VRML_SIM R2021a utf8
WorldInfo {
  coordinateSystem "NUE"
}
Viewpoint {
  orientation 0.006190388663951771 0.8703723550079406 0.4923551997553398 3.116611654332592
  position 0.2432714857456455 10.663007148506923 -8.090918802067446
}
TexturedBackground {
}
RectangleArena {
  rotation 0 1 0 4.692820414042842e-06
  floorSize 4 7
  wallHeight 0.5
}
CardboardBox {
  translation 0.51 0.16 -1.12
  size 0.3 0.3 0.3
}
DEF MY_ROBOT Create {
  translation 1.10386 0.044 -2.77248
  rotation 0 -1 0 -0.523595307179586
  name "Create1(1)"
  controller "left_robot"
  supervisor TRUE
  bodySlot [
    Compass {
      name "compass1"
    }
    DistanceSensor {
      translation -0.0881691 0.00965921 -0.15817
      rotation -0.13033599912122973 -0.9899989933250852 -0.05398629963600575 -2.3632753071795864
      name "left_dis_sensor"
      numberOfRays 4
      aperture 1.5707
    }
    DistanceSensor {
      translation 0.120389 0.00205559 -0.135355
      rotation -0.24128703593994796 -0.7761771156123661 -0.582520086766956 -0.8154053071795859
      name "right_dis_sensor"
      numberOfRays 4
      aperture 1.5707
    }
    Shape {
      geometry Cylinder {
        height 0.4
        radius 0.005
      }
    }
    Transform {
      translation 0 0.24 0
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 0.6 0.2 0.8
            }
          }
          geometry Sphere {
            radius 0.05
          }
        }
      ]
    }
    Camera {
      translation 0 0.04 0
      rotation 1 0 0 1.55
      name "CAM1"
      fieldOfView 3.14
      width 500
      height 500
      spherical TRUE
    }
  ]
}
DEF MY_ROBOT2 Create {
  translation -0.383863 0.044 -2.63966
  rotation 0 1 0 0.785394
  name "Create1(2)"
  controller "right_robot"
  supervisor TRUE
  bodySlot [
    Compass {
      name "compass2"
    }
    DistanceSensor {
      translation -0.0881691 0.00965921 -0.15817
      rotation -0.13033599912122973 -0.9899989933250852 -0.05398629963600575 -2.3632753071795864
      name "left_dis_sensor"
      numberOfRays 4
      aperture 1.5707
    }
    DistanceSensor {
      translation 0.120389 0.00205559 -0.135355
      rotation -0.24128703593994796 -0.7761771156123661 -0.582520086766956 -0.8154053071795859
      name "right_dis_sensor"
      numberOfRays 4
      aperture 1.5707
    }
    Shape {
      geometry Cylinder {
        height 0.4
        radius 0.005
      }
    }
    Transform {
      translation 0 0.24 0
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 0.6 0.2 0.8
            }
          }
          geometry Sphere {
            radius 0.05
          }
        }
      ]
    }
    Camera {
      translation 0 0.04 0
      rotation 1 0 0 1.55
      name "CAM2"
      fieldOfView 3.14
      width 500
      height 500
      spherical TRUE
    }
  ]
}
DEF Tracked Create {
  translation 0.028742 0.044 -1.88676
  rotation 0 1 0 -3.1415853071795863
  name "Create2"
  controller "nord_robot"
  bodySlot [
    DistanceSensor {
      translation -0.105605 0.02 -0.141716
      rotation 0.1304469958955108 0.9908389688234456 0.03495309890020758 2.62258
      scale 2 2 2
      name "left_dis_sensor"
      numberOfRays 4
      aperture 2
    }
    DistanceSensor {
      translation 0.0859809 0.02 -0.155
      rotation -0.12731306297803213 -0.9670394783660207 -0.2205131090813569 -1.0764653071795864
      scale 2 2 2
      name "right_dis_sensor"
      numberOfRays 4
      aperture 2
    }
    Shape {
      geometry Cylinder {
        height 0.4
        radius 0.005
      }
    }
    Transform {
      translation 0 0.24 0
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 0 0 0
            }
          }
          geometry Sphere {
            radius 0.05
          }
        }
      ]
    }
    Compass {
    }
    Camera {
      translation 0 0.14 0
      name "CAM3"
      fieldOfView 2
      width 1000
      height 350
    }
  ]
}
TexturedBackgroundLight {
}
